### ISSUE TITLE
avoid-operation-name-prefix - Cannot read property 'caseSensitive' of undefined

### ISSUE THREAD

**Issue by adrian-beachbody** [dotansimha/graphql-eslint#373]
**Describe the bug**
When applying the `@graphql-eslint/avoid-operation-name-prefix` rule in my `.eslintrc.js` config file. I see this error in my Code output console

```
[Error - 10:46:57 PM] ESLint stack trace:
[Error - 10:46:57 PM] TypeError: Cannot read property 'caseSensitive' of undefined
Occurred while linting /src/api/bff/mutations.graphql:1
    at verifyRule$1 (/node_modules/@graphql-eslint/eslint-plugin/index.cjs.js:540:46)
    at OperationDefinition (/node_modules/@graphql-eslint/eslint-plugin/index.cjs.js:622:17)
    at /node_modules/eslint/lib/linter/safe-emitter.js:45:58
    at Array.forEach (<anonymous>)
    at Object.emit (/node_modules/eslint/lib/linter/safe-emitter.js:45:38)
    at NodeEventGenerator.applySelector (/node_modules/eslint/lib/linter/node-event-generator.js:256:26)
    at NodeEventGenerator.applySelectors (/node_modules/eslint/lib/linter/node-event-generator.js:285:22)
    at NodeEventGenerator.enterNode (/node_modules/eslint/lib/linter/node-event-generator.js:299:14)
    at CodePathAnalyzer.enterNode (/node_modules/eslint/lib/linter/code-path-analysis/code-path-analyzer.js:711:23)
    at /node_modules/eslint/lib/linter/linter.js:954:32
```

**To Reproduce**

Run the `@graphql-eslint/avoid-operation-name-prefix` on these contents 
```graphql
mutation EnrollProgram(
  $programSlug: String!
  $enroll: Boolean!
  $datetime: String
) {
  enrollment(programSlug: $programSlug, enroll: $enroll, datetime: $datetime) {
    progress {
      enrollDate
    }
  }
}

```

**Expected behavior**
The rule does not throw an error

**Environment:**

- OS: MacOS Mojave 10.14.5
- `@graphql-eslint/plugin`: 0.9.1
- NodeJS:  v12.14.1

**Comment by dotansimha** [comment]
Fixed, we'll cut a release soon! :)

**Comment by dotansimha** [comment]
Fixed in `v1.0.0`! 🎉
